celestia.cfg

```diff
@@ -82,34 +82,34 @@ StarTextures
 {
 	# This texture will be used for any spectral type not listed
 	# in this block.
-	Default "astar.jpg"
+	Default "star-type-a.*"
 
-	O "bstar.*"
-	B "bstar.*"
-	A "astar.*"
-	F "astar.*"
-	G "gstar.*"
-	K "gstar.*"
-	M "mstar.*"
+	O "star-type-o.*"
+	B "star-type-b.*"
+	A "star-type-a.*"
```

Comment:
Why are the star textures renamed? Wouldn't it make more sense to keep the names from earlier, for the sake of compatibility?

Reply:
I don't know, those are FarGetaNik's names for them. The last release of Celestia Origin renamed their star textures in the same way. I don't think any of the old star textures are referenced in data files or addons, so there wouldn't be compatibility problems (note that I did keep the old browndwarf texture, for the sake of compatibility).

Reply:
you can rename them

Reply:
Give me a good reason why.

Reply:
Compatibility - I'm pretty sure there are older add-ons on the Motherlode that use "mstar.*", "gstar.*", etc.

Reply:
Like, way back when binary stars had to be defined as planets?

I prefer all the textures beginning with "star", but if everyone disagrees I'll rename them.

Reply:
Maybe, but I'm also talking about add-ons like [these](http://www.celestiamotherlode.net/catalog/show_addon_details.php?addon_id=1254) where they use the original naming format.

Reply:
*sigh*...I really don't feel like doing this right now.